src/main/groovy/life/qbic/datamodel/dtos/imaging/Image.groovy

```diff
@@ -101,4 +101,37 @@ final class Image {
     this.pixelSize = pixelSize
     this.resolution = resolution
   }
+
+  boolean equals(o) {
+    if (this.is(o)) return true
+    if (getClass() != o.class) return false
+
+    Image image = (Image) o
+
+    if (acquisitionDate != image.acquisitionDate) return false
+    if (acquisitionParameters != image.acquisitionParameters) return false
+    if (analysisParameters != image.analysisParameters) return false
+    if (imageId != image.imageId) return false
+    if (imageSize != image.imageSize) return false
+    if (instrument != image.instrument) return false
+    if (modality != image.modality) return false
+    if (pixelSize != image.pixelSize) return false
+    if (resolution != image.resolution) return false
+
+    return true
+  }
+
+  int hashCode() {
+    int result
+    result = (acquisitionDate != null ? acquisitionDate.hashCode() : 0)
```

Comment:
Elvis ftw
```suggestion
    result = ( acquisitionDate?.hashCode() ?: 0)
```

Reply:
I will do that in a separate commit. The solution above is the default intellij groovy hashCode method.